docs/CLI.md

```diff
@@ -111,44 +111,44 @@ In the validate command, if the catalog file is missing the `validate` command w
-npm run generate-example-output # Full command `npx ts-node src/opat.ts output -f tests/examples/valid.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml -o tests/examples/valid.markdown`
-npm run generate-drupal-output # Full command `npx ts-node src/opat.ts output -f opat/drupal-9.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml -o opat/drupal-9.markdown`
-npm run generate-example-html # Full command `npx ts-node src/opat.ts output -f tests/examples/valid.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml -o tests/examples/valid.html`
-npm run generate-drupal-html # Full command `npx ts-node src/opat.ts output -f opat/drupal-9.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml -o opat/drupal-9.html`
+npm run generate-example-output # Full command `npx ts-node src/openacr.ts output -f tests/examples/valid.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml -o tests/examples/valid.markdown`
+npm run generate-drupal-output # Full command `npx ts-node src/openacr.ts output -f openacr/drupal-9.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml -o openacr/drupal-9.markdown`
+npm run generate-example-html # Full command `npx ts-node src/openacr.ts output -f tests/examples/valid.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml -o tests/examples/valid.html`
+npm run generate-drupal-html # Full command `npx ts-node src/openacr.ts output -f openacr/drupal-9.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml -o openacr/drupal-9.html`
 ```
 
-We also have a GitHub action called 'Drupal 9 OPAT output' that will generate the markdown and HTML versions of the Drupal 9 OPAT. It is run on pull requests, and the output can be downloaded to double-check it is matching expectations.
+We also have a GitHub action called 'Drupal 9 OpenACRs output' that will generate the markdown and HTML versions of the Drupal 9 OpenACRs. It is run on pull requests, and the output can be downloaded to double-check it is matching expectations.
 
 The tests also generates output that is stored in the `output` directory but is not tracked by git.
 
 ### USWDS Design System and stylesheets
 
 The HTML output comes with and uses styles from the [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/).
 
-The output also includes two stylesheets that provide additional styling for the OPAT:
+The output also includes two stylesheets that provide additional styling for the OpenACRs:
 
-- `opat.css` has customization on top of the USWDS design system.
+- `openacr.css` has customization on top of the USWDS design system.
 - `custom.css` provided to add any additional customizations.
 
-## OPATs
+## OpenACRs
 
-Current example OPATs that are tracked in this repository are in the `opat` directory. The directory includes a data table representation of all the OPATs with the ability to search them.
+Current example OpenACRs that are tracked in this repository are in the `opopenacrat` directory. The directory includes a data table representation of all the OpenACRs with the ability to search them.
```

Comment:
Noticed typo `opopenacrat`

Reply:
I kinda like the sound of opopenacrat...  :)